GitHubRepositories.ps1

```diff
@@ -415,11 +417,12 @@ function Get-GitHubRepository
         [Parameter(ParameterSetName='Organization')]
         [string] $OrganizationName,
 
-        [ValidateSet('All', 'Public', 'Private')]
         [Parameter(ParameterSetName='AuthenticatedUser')]
+        [ValidateSet('All', 'Public', 'Private')]
         [string] $Visibility,
 
         [Parameter(ParameterSetName='AuthenticatedUser')]
+        [ValidateSet('Owner', 'Collaborator', 'Organization_Member')]
```

Comment:
From a PowerShell perspective, this should be:

```suggestion
        [ValidateSet('Owner', 'Collaborator', 'OrganizationMember')]
```

That would mean you'd need a parameter value translator hashtable before assigning the actual value in the API request

Reply:
Done